packages/apollo-forest-run/src/cache/env.ts

```diff
@@ -35,6 +34,9 @@ export function createCacheEnvironment(config?: CacheConfig): CacheEnv {
     },
     mergePolicies: new Map(),
     readPolicies: new Map(),
+    autoEvict: config?.autoEvict ?? true,
```

Comment:
nitpick: Would be nice to have it sorted alphabetically someday

Reply:
Maybe someday :) I prefer having related options grouped together - so if we sort alphabetically we will need to rename them to have a common prefix for the same "group" (or introduce another level of objects for grouping). Which is a breaking change.